From 8a7a8e1eab929eb3a5b735a788a23b9731139046 Mon Sep 17 00:00:00 2001
From: Dou Liyang <douly.fnst@cn.fujitsu.com>
Date: Mon, 13 Nov 2017 13:49:04 +0800
Subject: [PATCH] timekeeping: Eliminate the stale declaration of
 ktime_get_raw_and_real_ts64()
Git-commit: 8a7a8e1eab929eb3a5b735a788a23b9731139046
Patch-mainline: v4.15-rc1
References: bsc#1099918

Commit ba26621e63ce got rid of ktime_get_raw_and_real_ts64(), but left its
declaration behind.

Remove it.

Fixes: ba26621e63ce ("time: Remove duplicated code in ktime_get_raw_and_real()")
Signed-off-by: Dou Liyang <douly.fnst@cn.fujitsu.com>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Cc: Christopher S. Hall <christopher.s.hall@intel.com>
Cc: joelaf@google.com
Cc: arnd@arndb.de
Cc: gregkh@linuxfoundation.org
Cc: john.stultz@linaro.org
Cc: deepa.kernel@gmail.com
Cc: stable@vger.kernel.org
Link: https://lkml.kernel.org/r/1510552144-20831-1-git-send-email-douly.fnst@cn.fujitsu.com
Signed-off-by: Coly Li <colyli@suse.de>

---
 include/linux/timekeeping.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/include/linux/timekeeping.h b/include/linux/timekeeping.h
index 0021575fe871..51293e1aa4da 100644
--- a/include/linux/timekeeping.h
+++ b/include/linux/timekeeping.h
@@ -272,12 +272,6 @@ extern bool timekeeping_rtc_skipresume(void);
 
 extern void timekeeping_inject_sleeptime64(struct timespec64 *delta);
 
-/*
- * PPS accessor
- */
-extern void ktime_get_raw_and_real_ts64(struct timespec64 *ts_raw,
-				        struct timespec64 *ts_real);
-
 /*
  * struct system_time_snapshot - simultaneous raw/real time capture with
  *	counter value
-- 
2.17.1

